src/odemis/acq/align/light.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
```

Comment:
Looks like the right place/name now :-) However, don't forget to remove the other file (turnOnLight.py), and the original test cases, (in align_test.py).